### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by basil**
Another series of code cleanup changes for this repository. Most of these changes fix IntelliJ IDEA warnings. I kept each commit separate. Each commit message should be self-explanatory, but if anything is unclear I am happy to provide further explanation. Also, feel free to revert any specific cleanup commits that are unwanted.

### CODE REVIEW

**Review by res0nance — APPROVED** [review]

**Review by StefanSpieker — APPROVED** [review]

File: core/src/main/java/hudson/util/RingBufferLogHandler.java [hunk 1]
```diff
@@ -23,9 +23,6 @@
  */
 package hudson.util;
 
-import org.kohsuke.accmod.Restricted;
```
Comment (timja) [inline core/src/main/java/hudson/util/RingBufferLogHandler.java:26]: can we add a checkstyle rule to prevent re-occurrence please?

Comment (basil) [reply core/src/main/java/hudson/util/RingBufferLogHandler.java:26]: Declined, the scope of this PR is fixing existing violations, not adding automated enforcement. Automated enforcement can be done in a separate PR.

File: test/src/test/java/lib/form/TextAreaTest.java [hunk 2]
```diff
@@ -72,7 +72,7 @@ public FormValidation doCheckText2(@QueryParameter String text1) {
     @Issue("JENKINS-27505")
     @Test
     public void text() throws Exception {
-        T1: {
+        {
```
Comment (oleg-nenashev) [inline test/src/test/java/lib/form/TextAreaTest.java:75]: A proper refactoring would be to split it to separate tests. 
No string opinion about deletion, we have comments anyway

**Review by oleg-nenashev — APPROVED** [review]
All looks good, thanks!

We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Pull request merged**